tests/tck/features/yield/parameter.feature

```diff
@@ -209,3 +266,11 @@ Feature: Parameter
       """
     Then a SyntaxError should be raised at runtime: Direct output of variable is prohibited near `$var'
     Then clear the used parameters
+
```

Comment:
Please add some test scenarios where parameter expressions and variable expressions are mixed.

Reply:
```
 $var=go from "Tim Duncan" over like yield like._dst as id
 | yield $-.id+$p1+count(distinct abs($p1)+$-.id)
```

Reply:
done